app/assets/stylesheets/publisher.scss

```diff
@@ -243,14 +237,6 @@
     .hide-location { display: inline-block; }
     .locator { display: none; }
   }
-
-  .twitter-typeahead {
-    left: -1px;
-    // Override inline rule of Typeahead
-    // scss-lint:disable ImportantRule
-    position: absolute !important;
```

Comment:
Please keep `position: absolute` for the publisher.

![publisher relative typeahead](https://cloud.githubusercontent.com/assets/3798614/24244923/2a3701fc-0fc1-11e7-9a7a-d8223c1cc03c.png)

Having `position: relative` everywhere else isn't perfect but I wasn't able to find a better solution.